apps/web/app/meetings/_components/Reservation/ReservationForm.tsx

```diff
@@ -0,0 +1,200 @@
+              <Dropdown
+                selectedValue={field.value}
+                onSelect={(value: string | boolean) => {
+                  field.onChange(value);
+                }}
+                isError={Boolean(errors.endTime)}
+                errorMessage={errors.endTime?.message ?? ""}
+              >
+                <Dropdown.Toggle title="종료 시간">{field.value || "종료 시간 선택"}</Dropdown.Toggle>
+                <Dropdown.Wrapper>
+                  <Dropdown.Item value="custom-end">직접입력</Dropdown.Item>
+                  {timeOptions.map((time) => (
+                    <Dropdown.Item key={time} value={time}>
+                      {time}
+                    </Dropdown.Item>
+                  ))}
+                </Dropdown.Wrapper>
+              </Dropdown>
+            )}
+          />
+          {endTimeValue === "custom-end" && (
+            <Controller
+              name="customEndTime"
+              control={control}
+              rules={{ required: "종료 시간을 입력해주세요." }}
+              render={({ field }) => <Input id="custom-end-time" placeholder="종료 시간 (HH:MM)" {...field} />}
+            />
+          )}
+          {errors.customEndTime ? <p className="text-red-500">{errors.customEndTime.message}</p> : null}
+        </div>
+      </div>
+
+      <Controller
+        name="participants"
+        control={control}
+        render={({ field }) => (
+          <MultiSelectDropdown
+            selectedValue={field.value}
+            onSelect={(value: string[]) => {
+              field.onChange(value);
+            }}
+          >
+            <MultiSelectDropdown.Toggle title="참여자">
+              {field.value.length > 0 ? (
+                <div className="flex flex-wrap gap-2">
+                  {field.value.slice(0, 4).map((name) => (
+                    <Profile key={name} size="size-27" name={name} />
+                  ))}
+                  {field.value.length > 4 && <span>+{field.value.length - 4}명 더보기</span>}
+                </div>
+              ) : (
+                "참여자 선택"
+              )}
+            </MultiSelectDropdown.Toggle>
+            <MultiSelectDropdown.Wrapper>
+              {mockParticipants
+                .sort((a, b) => {
+                  const isASelected = field.value.includes(a);
+                  const isBSelected = field.value.includes(b);
+
+                  if (isASelected && !isBSelected) return -1;
+                  if (!isASelected && isBSelected) return 1;
+                  return 0;
+                })
+                .map((name) => (
+                  <MultiSelectDropdown.Item key={name} value={name}>
+                    <Profile name={name} size="size-27" />
+                  </MultiSelectDropdown.Item>
+                ))}
+            </MultiSelectDropdown.Wrapper>
+          </MultiSelectDropdown>
+        )}
+      />
+
+      <Button variant="Primary" className="mt-4 w-full" onClick={() => void handleSubmit(onSubmit)()}>
+        예약하기
+      </Button>
```

Comment:
[대충 관련 아티클](https://medium.com/@edidee/using-onclick-vs-onsubmit-to-submit-a-form-in-react-c7e6dd318dce)
글 한번 읽어보고 넘어갔으면 좋겠어!

For form submission, the onClick event is not the best practice. This is because it doesn’t account for keyboard events like pressing the enter button. Users must always click on the button before a form can be submitted. It's not a good user experience if you ask me.

아마 onClick으로 처리하면 엔터와 같은 키입력은 인식하지 못하니까 form태그에서의 onSubmit이 조금 더 UX차원에서 사용하기 좋을거야